src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -915,4 +915,42 @@ DocumentUpload findDocumentUploadInEvent(final ProcurementEvent event, final Str
             String.format(ERR_MSG_FMT_DOCUMENT_NOT_FOUND, documentId)));
   }
 
+  /**
+   * Export buyer attachments
+   * 
+   * @param procId
+   * @param eventId
+   * @param principal
+   * @return list of attachments
+   */
+  public List<DocumentAttachment> exportDocuments(final Integer procId, final String eventId,
```

Comment:
This method needs `@Transactional` on it to avoid a Hibernate lazy loading error around the document collection (I just hit that when doing an export).